docs/quick-visual-tutorials/visual-k8s-cluster-mapping.mdx

```diff
@@ -143,17 +143,15 @@ We also (as a default) told Otterize Cloud that there is a global default-deny n
 
 #### Blocking status
 
-Note that the locks themselves are green, indicating that you could now turn on enforcement without blocking any clients.
+Note that the locks themselves are yellow, indicating that you could now turn on enforcement and blocking not intented clients.
 
-Similarly, all the thick connecting lines between the services are green: none of these client calls would be blocked if enforcement were turned on. If one were red, that would tell you it would be blocked, as you might have guessed.
+Similarly, all the thick connecting lines between the services are yellow: Client calls would be blocked if enforcement were turned on. If one were red, that would tell you it is blocked, as you might have guessed.
```

Comment:
```suggestion
Similarly, all the connecting lines between the services are yellow: Client calls would be blocked if the services were protected using a `ProtectedService` resource, however would not be blocked if enforcement were merely on. Click any line to see this.
```

Reply:
@sapirwo Can you add a screenshot here to show the explanation on the edge, showing that if the service is protected, then it would block?